homeassistant/components/sensor/fido.py

```diff
@@ -50,12 +50,12 @@
     'text_int_used': ['International text used',
                       MESSAGES, 'mdi:message-alert'],
     'text_int_limit': ['International text limit',
-                       MESSAGES, 'mdi:message-alart'],
-    'text_int_remaining': ['Internaltional remaining',
+                       MESSAGES, 'mdi:message-alert'],
+    'text_int_remaining': ['International remaining',
                            MESSAGES, 'mdi:message-alert'],
     'talk_used': ['Talk used', MINUTES, 'mdi:cellphone'],
     'talk_limit': ['Talk limit', MINUTES, 'mdi:cellphone'],
-    'talt_remaining': ['Talk remaining', MINUTES, 'mdi:cellphone'],
+    'talk_remaining': ['Talk remaining', MINUTES, 'mdi:cellphone'],
```

Comment:
This needs to be reviewed by someone with knowledge of and access to this sensor. Maybe this is just a spelling mistake of the API.

Reply:
Ok, seems to be a simple spelling mistake. See https://github.com/titilambert/pyfido/blob/52723bcb8b1a445789926c3ae15d60f6658e7359/pyfido/__main__.py#L31